packages/dd-trace/src/appsec/index.js

```diff
@@ -159,6 +159,7 @@ function disable () {
   isEnabled = false
   config = null
 
+  resetTemplates()
```

Comment:
Why do we need it?

Reply:
There are some tests that expect to load several templates but with the addition of the boolean they didn't pass because, once the initial ones are loaded, there is no way to load a different one. I thought that, rather than removing that tests, it could be useful have the ability to load different ones after the appsec module is disabled and enabled again.

Reply:
But at this way, if the appsec is disabled and enabled again using remote configuration. It will reload the templates unnecessarily. Should we call this method only from the test?